pyproject.toml

```
@@ -1,5 +1,138 @@
 [build-system]
-requires = ["setuptools",
-            "wheel",
-            "setuptools_scm"]
-build-backend = 'setuptools.build_meta'
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "stellarphot"
+dynamic = ["version"]
+description = "A package for performing stellar photometry."
+readme = "README.md"
+license = { file = "LICENSE.rst" }
```

Oh, one other cool thing if you want is that with Hatchling this can be defined as a string because we were very early adopters of the future proposal that was always planned https://hatch.pypa.io/latest/config/metadata/#license

Don't worry about it not being accepted, the efforts to apply the final feedback for acceptance has recommenced: https://discuss.python.org/t/pep-639-round-2-improving-license-clarity-with-better-package-metadata/12622/96

The person was busy for some time working on space stuff 🙂 https://discuss.python.org/t/pep-639-round-2-improving-license-clarity-with-better-package-metadata/12622/81

```suggestion
license = "BSD-3-Clause"
```